ERCS/erc-7858.md

```diff
@@ -94,14 +95,14 @@ interface IERC7858 /**is IERC721 */ {
 - `balanceOf` that inherited from [ERC-721](./eip-721.md) MUST return all tokens even if expired; they still exist but are unusable due to the limitation of tracking expired token on-chain.
 - For NFTs `transferFrom`, and `safeTransferFrom` MUST allow transferring tokens even if they expired. This ensures that expired tokens remain transferable and tradable, preserving compatibility with existing applications already deployed. However, expired tokens MUST be considered invalid and unusable in contracts that check for token validity.
 - `expiryType` MUST return the type of expiry used by the contract, which can be either `BLOCK` or `TIME`.
-- `isTokenExpired` is used for retrieving the status of the given `tokenId`; the function MUST return `true` if the token is expired and MUST revert if the `tokenId` does not exist for implementation that use custom error SHOULD revert with `ERC721NonexistentToken` following the relevant [ERC-6093](./eip-6093.md) and for implementation that using Solidity version below `v0.8.4` or those preferring to use revert with string error SHOULD revert with `NonexistToken`, If the `tokenId` exists and is not expired, the function MUST return `false`.
+- `isTokenExpired` is used for retrieving the status of the given `tokenId`; the function MUST return `true` if the token is expired and MUST revert if the `tokenId` does not exist. Implementations that use custom errors SHOULD revert with `ERC721NonexistentToken` following the relevant [ERC-6093](./eip-6093.md) and implementations using Solidity version below `v0.8.4` or those preferring to use revert with string error SHOULD revert with `NonexistToken`. If the `tokenId` exists and is not expired, the function MUST return `false`.
 - `startTime` and `endTime` of `tokenId`, can be `block.number` or `block.timestamp` depending on `expiryType`. The `startTime` MUST less than `endTime` and SHOULD except when both are set to `0`. A `startTime` and `endTime` of `0` indicates that the `tokenId` has no time-limited. In case `startTime` is non `0` but the `endTime` is `0` also considered that `tokenId` has no time-limited. If the `tokenId` does not exist MUST revert same as in function `isTokenExpired`.
 - `supportInterface` for `IERC7858` is `0x3ebdfa31` for `IERC7858Epoch` is `0xec7ffd66`
```

Comment:
How about:

```suggestion
- The interface ID for [ERC-165](./eip-165.md)'s `supportsInterface` for `IERC7858` is `0x3ebdfa31`, and for `IERC7858Epoch` it is `0xec7ffd66`
```

Also, make sure to double check these before we merge this PR!